spec/factories/labware_types.rb

```diff
@@ -1,43 +1,43 @@
 FactoryBot.define do
 
   factory :labware_type do
-    num_of_cols 12
-    num_of_rows 8
-    col_is_alpha false
-    row_is_alpha false
+    num_of_cols { 12 }
```

Comment:
Why are we defining blocks for these variables?

Reply:
I had some problems while running the tests that for some reason were solved after changing this to blocks

Reply:
Haha, weird!

Reply:
It may be possible that after updating some libraries it was in a version of factorybot that deprecated this (possibly as they say here  https://robots.thoughtbot.com/deprecating-static-attributes-in-factory_bot-4-11).  
It was in commit: 4f264c8bef251a5ee5930ebb8d6e645f1f58786a

Reply:
👍